src/detectors/code_injection/code_injection.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// {fact rule=code-injection@v1.0 defects=1}
+var express = require("express")
+var app = express()
+
+var { spawn } = require("child_process")
+function codeInjectionNoncompliant() {
+    app.get("/:command/:params", (req, res) => {
+        // Noncompliant: user-supplied parameters are evaluated as a script.
+        const subProcess = spawn(req.params.command, [req.params.params])
```

Comment:
This looks like an example of OS Command Injection to me, rather than a Code Injection one.

See https://cwe.mitre.org/data/definitions/78.html vs https://cwe.mitre.org/data/definitions/94.html

Reply:
Changed the examples.